### PR TITLE
Can now build on GHC 7.10

### LINKED ISSUE #23: GHC 7.10.1 fails to install
Installed haddock-docs
    cabal: InstallPlan: internal error: configured package depends on a non-library package

cabal version 1.22.2.0
cabal-install version 1.22.4.0

### PR DESCRIPTION
**Pull request by ivan-m**
Closes #23

### CODE REVIEW

File: src/Haskell/Docs/Cabal.hs [hunk 1]
```diff
@@ -29,22 +33,16 @@ getGhcOpsPackageDB gs = map (SpecificPackageDB . trim) pkgDBOps
 
 -- | Get all installed packages, filtering out the given package.
 getAllPackages :: [String] -> Ghc [PackageConfig.PackageConfig]
-getAllPackages gs =
+getAllPackages _gs =
   do flags <- getSessionDynFlags
      return (fromMaybe [] (pkgDatabase flags))
-  -- do config <- configureAllKnownPrograms
```
Comment (chrisdone) [inline src/Haskell/Docs/Cabal.hs]: Why's this removed?

Comment (ivan-m) [inline src/Haskell/Docs/Cabal.hs]: It was already commented out, so I just removed the useless comment.

I can put that comment back in if you'd prefer.

Comment (chrisdone) [inline src/Haskell/Docs/Cabal.hs]: Oh fair enough. Seems fine then.

**Comment by chrisdone** [comment]
Cheers!

**Comment by ivan-m** [comment]
Well, I got sick of waiting ;-)

**Pull request merged**